acid/features/auth/service.py

```diff
@@ -52,7 +50,8 @@ def fetch_user_data():
 
 def start_openid_auth():
     oid_consumer = consumer.Consumer(session, None)
-    oid_request = oid_consumer.begin(config['default']['openid_provider'])
+    oid_request = oid_consumer.begin(
```

Comment:
service shouldn't know about `current_app` and config at all. `openid_provider` must be pass as argument.